SmartDeviceLink/SDLVideoStreamingCapability.h

```diff
@@ -0,0 +1,50 @@
+//
+//  SDLVideoStreamingCapability.h
+//  SmartDeviceLink-iOS
+//
+//  Created by Brett McIsaac on 7/31/17.
+//  Copyright © 2017 smartdevicelink. All rights reserved.
+//
+
+#import "SDLRPCMessage.h"
+
+@class SDLImageResolution;
+@class SDLVideoStreamingFormat;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLVideoStreamingCapability : SDLRPCStruct
+
+/**
+ * @abstract Constructs a newly allocated SDLVideoStreamingCapability object
+ */
+- (instancetype)init;
+
+/**
+ * @abstract Constructs a newly allocated SDLVideoStreamingCapability object indicated by the dictionary parameter
+ * @param dict The dictionary to use
+ */
+- (instancetype)initWithDictionary:(NSMutableDictionary *)dict;
```

Comment:
If you check other `SDLRPCStruct` subclasses, you'll see that the interfaces and implementations for `init` and `initWithDictionary` are removed in v5.0.